etc/db-config.yaml

```
@@ -223,7 +223,7 @@
             description: Print times of contest events relative to contest start (instead of absolute).
         -   name: time_format
             type: string
-            default_value: "%H:%M"
+            default_value: "H:i"
             public: false
             description: The format used to print times. For formatting options see the PHP 'strftime' function.
```

```suggestion
            description: The format used to print times. For formatting options see the PHP 'date' function.
```

Not sure if this is the correct suggestion, but you implied some formatting has changed.

---

Yup. Thanks